integration_tests/e2e/support-needs.cy.ts

```diff
@@ -0,0 +1,199 @@
+      .first()
+      .check()
+    // Tick 'Other' and fill in custom support need.
+    cy.get('.govuk-checkboxes__input[name="support-need-option-Accommodation related debt and arrears"]').eq(2).check()
+    cy.get('.govuk-input').eq(2).type('Custom accommodation support need')
+
+    // Click Continue
+    cy.get('.govuk-button[type="submit"]').click()
+    cy.get('.govuk-grid-column-three-quarters > h1').should('contain.text', 'End a tenancy')
+
+    // Test back button
+    cy.get('.govuk-back-link').click()
+    cy.get('.govuk-grid-column-full > h1').should('contain.text', 'Add accommodation support needs')
+    cy.get('.govuk-button[type="submit"]').click()
+
+    // Fill in 'End a tenancy' support need
+    cy.get('[id="status"]').check()
+    cy.get('[id="responsibleStaff"]').check()
+    cy.get('.govuk-textarea[id="updateText"]').type('Some additional details')
+    cy.get('.govuk-button[type="submit"]').first().click()
+
+    // Remove support need 'Set up payment for rent arrears'
+    cy.get('.ghost-button').first().click()
+    cy.get('.confirm-remove-support-need .govuk-button[type="submit"]').click()
+
+    // Fill in 'Custom accommodation support need'
+    cy.get('.govuk-grid-column-three-quarters > h1').should('contain.text', 'Custom accommodation support need')
+    cy.get('[id="status-2"]').check()
+    cy.get('[id="responsibleStaff-2"]').check()
+    cy.get('.govuk-button[type="submit"]').first().click()
+
+    // Submit 'Check your answers' page
+    cy.get('.govuk-grid-column-two-thirds-from-desktop > h1').should('contain.text', 'Check your answers')
+    cy.get('.govuk-grid-column-two-thirds-from-desktop > h2').should('contain.text', 'Accommodation support needs')
+    cy.get('.govuk-button[type="submit"]').first().click()
+
+    // Should be back to the accommodation overview page
+    cy.get('.govuk-grid-column-two-thirds > h2').should('contain.text', 'Accommodation')
+    cy.get('.app-summary-card__title').should('contain.text', 'Support needs')
+  })
+
+  it('Editing an existing Accommodation support need', () => {
+    cy.task('stubJohnSmithGetAccommodationWithSupportNeeds')
+
+    cy.signIn()
+
+    cy.visit('/accommodation/?prisonerNumber=A8731DY')
+
+    cy.get('.app-summary-card__title').should('contain.text', 'Support needs')
+
+    cy.get('.govuk-summary-card__actions').should('have.attr', 'href')
+    cy.get('.govuk-summary-card__actions').should('contain.text', 'Add a support need')
+
+    // Click to edit 'End a tenancy' support need
+    cy.get('tr td:first > a').should('have.attr', 'href')
+    cy.get('tr td:first > a').should('contain.text', 'End a tenancy').click()
+
+    // Update 'End a tenancy' fields
+    cy.get('.govuk-heading-l').should('contain.text', 'End a tenancy')
+    cy.get('[id="updateStatus-2"]').check()
+    cy.get('[id="responsibleStaff-2"]').check()
+    cy.get('.govuk-textarea[id="additionalDetails"]').type('Update')
```

Comment:
Wondering - is there more than one element on the page with that ID?

If so we need to update the FE - ids should be unique on a page (it's ok to have the same ID used across many pages, but not more than once per page).

If not, can we simply the selector by just making it a get by id?